pkg/bootstrap/testdata/authsds_golden.json

```diff
@@ -287,7 +290,7 @@
               "endpoint": {
                 "address":{
                   "pipe": {
-                    "path": "/etc/istio/proxy/SDS"
```

Comment:
Why do we need ".", is there anything broken without "."?

Reply:
yes you cannot run without write access to /etc (generally root). which breaks VMs/local running

Reply:
Thanks.